core/src/test/scala/org/apache/spark/deploy/JsonProtocolSuite.scala

```diff
@@ -25,24 +25,11 @@ import org.json4s._
 import org.json4s.jackson.JsonMethods
 import org.scalatest.FunSuite
 
-import org.apache.spark.deploy.DeployMessages.{MasterStateResponse, WorkerStateResponse}
-import org.apache.spark.deploy.master.{ApplicationInfo, DriverInfo, RecoveryState, WorkerInfo}
+import org.apache.spark.{JsonTestUtils, SparkConf}
+import org.apache.spark.deploy.DeployMessages.WorkerStateResponse
 import org.apache.spark.deploy.worker.{DriverRunner, ExecutorRunner}
-import org.apache.spark.SparkConf
 
-class JsonProtocolSuite extends FunSuite {
-
-  test("writeApplicationInfo") {
```

Comment:
With the removal of these tests, I've noticed that a number of the JSON responses in `JsonConstants` are now unused.  Do we have tests to ensure that the new JSON APIs implemented in this patch remains backwards-compatible with the old JSON written by our legacy implementation?  If not, we might want to add this because folks might be relying on the existing API.

Maybe this is already covered by the "expected responses" added elsewhere in this PR.  If so, then I think we should continue purging unused code from this file.